doc/nrf/security/ap_protect.rst

```diff
@@ -197,16 +209,80 @@ This is the default setting in the |NCS|.
 
 You can start debugging the firmware without additional steps needed.
 
-Once you are done debugging, run the following command to enable the access port protection:
+.. _app_approtect_uicr_approtect:
+
+Enabling AP-Protect by locking the UICR.APPROTECT
+=================================================
+
+For the devices in production, you are highly recommended to lock the UICR.APPROTECT register to prevent unauthorized access to the device. The access port protection configured in this way cannot be disabled without erasing the flash.
+
+.. note::
+    This is the only mechanism supported for the nRF52 series and nRF9160 devices that do not have firmware support for AP-Protect.
```

Comment:
```suggestion
    This is the only mechanism supported for the nRF52 series and nRF9160 devices that do not have software support for AP-Protect.
```

firmware->software as there is no FW running in nRF91 or nRF52 before the customer programs it with their own FW. The HW documentation talks about software support when locking the register.